src/search/view/SearchView.ts

```diff
@@ -55,12 +55,16 @@ import { IconButton } from "../../gui/base/IconButton.js"
 import { ButtonSize } from "../../gui/base/ButtonSize.js"
 import { BottomNav } from "../../gui/nav/BottomNav.js"
 import { MobileMailActionBar } from "../../mail/view/MobileMailActionBar.js"
+import { DrawerMenuAttrs } from "../../gui/nav/DrawerMenu.js"
+import { BaseTopLevelView } from "../../gui/BaseTopLevelView.js"
 
 assertMainOrNode()
 
-export class SearchView implements CurrentView {
-	view: CurrentView["view"]
-	oncreate: CurrentView["oncreate"]
+export interface SearchViewAttrs extends TopLevelAttrs {
+	drawerAttrs: DrawerMenuAttrs
+}
+
+export class SearchView extends BaseTopLevelView implements CurrentView<SearchViewAttrs> {
```

Comment:
Checked, basically there's no need to cache. Switching back to search does not work that well most of the time (e.g.: on mobile if I open a contact in search, then switch away and back to search it will show empty viewer) and we are better off re-loading the search result anyway (it is cached in searchModel so no actual search will be made again).